staging/src/k8s.io/apimachinery/pkg/labels/labels_test.go

```diff
@@ -54,7 +54,7 @@ func TestLabelHas(t *testing.T) {
 
 func TestLabelGet(t *testing.T) {
 	ls := Set{"x": "y"}
-	if ls.Get("x") != "y" {
+	if val, _ := ls.Get("x"); val != "y" {
```

Comment:
this is unnecessary when the second return is not used

Reply:
Hi, ls.Get('Key') is not map original func, it be wrapped in 
```
// Get returns the value and whether the provided label exists in the map.
func (ls Set) Get(label string) (string, bool) {	
       ele, exists := ls[label]	
       return ele, exists
}
```

Reply:
i see.